.env.dist

```diff
@@ -38,3 +38,9 @@ QUEUE_HOST=adshares_ads-operator_queue_1
 QUEUE_PORT=5672
 QUEUE_USER=guest
 QUEUE_PASSWORD=guest
+
+MAILER_URL=null://localhost
```

Comment:
`null`?

Reply:
Yep, default solution to disable email delivery. It was created by Symfony Flex. It has to be changed (ALWAYS) on the prod.

Reply:
https://symfony.com/doc/current/email.html